tetris.py

```diff
@@ -51,14 +51,18 @@ def run_gui():
     )
     topbar_score.pack(side="left", fill="x", expand=True)
 
+    topbar_holder = tkinter.Frame(topbar, bg=D_GREY, relief='ridge', height=square_size*3, width=square_size*5)
+    topbar_holder.pack(side='right', expand=True)
+    topbar_holder.pack_propagate(0)
```

Comment:
What does this do? Is it necessary?

Reply:
Added a comment explaining its role.

Reply:
I was referring to `pack_propagate(0)`. [The manual](https://tcl.tk/man/tcl8.6/TkCmd/pack.htm) says that it disables propagation and "the  packer  will not set the requested width and height of the packer", but I don't really get it.

Reply:
pack_propagate is by default set to 1 and makes the frame stretch in order to accomodate its windows. Setting it to 0 means height= and width= will stop being overlooked

Reply:
Then maybe:

```
topbar_holder.pack_propagate(0)   # don't overlook width and height
```